databricks/koalas/plot.py

```diff
@@ -1118,6 +1120,104 @@ def _plot(data, x=None, y=None, subplots=False, ax=None, kind="line", **kwds):
     return plot_obj.result
 
 
+def _find_backend(backend):
+    """
+    Find a koalas plotting backend
+    """
+    # function copied from pandas.plotting._core
+
+    # Delay import for performance.
+    import pkg_resources
+
+    for entry_point in pkg_resources.iter_entry_points("koalas_plotting_backends"):
+        if entry_point.name == "matplotlib":
+            # matplotlib is an optional dependency. When
+            # missing, this would raise.
+            continue
+        _backends[entry_point.name] = entry_point.load()
+
+    try:
+        return _backends[backend]
+    except KeyError:
+        try:
+            module = importlib.import_module(backend)
+        except ImportError:
+            # We re-raise later on.
+            pass
+        else:
+            if hasattr(module, "plot"):
+                # Validate that the interface is implemented when the option
+                # is set, rather than at plot time.
+                _backends[backend] = module
+                return module
+
+    # if failed to load plotting backend, set it back to default
+    from databricks.koalas import set_option
+
+    set_option("plotting.backend", "matplotlib")
+
```

Comment:
I don't think we should do this automatically.

Reply:
`Pandas` does it, I will remove it if its better that way.

Reply:
Seems like pandas throws an error when setting the option.

Reply:
Shall we remove this for now?